tslint.json

```diff
@@ -0,0 +1,100 @@
+{
+    "defaultSeverity": "error",
+    "extends": "typestrict",
+    "jsRules": {},
+    "rules": {
+        "arrow-parens": true,
+        "ban-ts-ignore": true,
+        "import-spacing": true,
+        "indent": {
+            "options": [
+                "spaces",
+                2
+            ]
+        },
+        "max-line-length": {
+            "options": {
+                "limit": 120
+            }
+        },
+        "no-debugger": true,
+        "no-console": true,
+        "new-parens": true,
+        "only-arrow-functions": {
```

Comment:
Prefer terser arrow function syntax where:
- there no are adverse effects (i.e. `this` not used within a `function` style function)
- function is not a declaration, e.g. `declare function foo(): void;`
- function is not named. e.g. `export function foo(): void { ... }`